From 346f4b1ec22136f65bfed4c5a9a3364c36734b58 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:05:57 -0400
Subject: [usb] core: Fix usb_free_streams return value documentation

Message-id: <1410887463-20674-286-git-send-email-dzickus@redhat.com>
Patchwork-id: 95423
O-Subject: [RHEL7 PATCH 285/591] usb-core: Fix usb_free_streams return value documentation
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 12d4bbcea727710bbd04de3e1de05957a0675e60
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Wed Oct 9 17:19:23 2013 +0200

    usb-core: Fix usb_free_streams return value documentation

    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hcd.c b/drivers/usb/core/hcd.c
index 674d2df..7fb817c 100644
--- a/drivers/usb/core/hcd.c
+++ b/drivers/usb/core/hcd.c
@@ -2078,8 +2078,7 @@ EXPORT_SYMBOL_GPL(usb_alloc_streams);
  * Reverts a group of bulk endpoints back to not using stream IDs.
  * Can fail if we are given bad arguments, or HCD is broken.
  *
- * Return: On success, the number of allocated streams. On failure, a negative
- * error code.
+ * Return: 0 on success. On failure, a negative error code.
  */
 int usb_free_streams(struct usb_interface *interface,
   struct usb_host_endpoint **eps, unsigned int num_eps,
-- 
1.7.1